lib/entities/app-action.ts

```diff
@@ -130,6 +138,14 @@ type LegacyFunctionAppActionProps = Record<string, unknown> & {
 export type CreateAppActionProps = AppActionCategory & {
   name: string
   description?: string
+  /**
+   * Optional JSON Schema describing the request payload shape
+   */
+  parametersSchema?: object
+  /**
+   * Optional JSON Schema describing the result shape
+   */
+  resultSchema?: object
```

Comment:
object includes arrays, functions, dates, regexes, so let's scope this down to more of what a JSONSchema would look like
```suggestion
  parametersSchema?: Record<string, unknown>
  /**
   * Optional JSON Schema describing the result shape
   */
  resultSchema?: Record<string, unknown>
```